### ISSUE TITLE
Card- consider refactoring markup for clickable cards

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#10188]
**Describe the problem**
When tabbing through the clickable cards, it almost appears as if the second card doesn't receive focus. It looks like the clickable card is performing the action on focus AND on click. It seems like it's clicking the link on focus which is why focus shifts off it so fast. This would be worth some investigation.

**How do you reproduce the problem?**
Go to[ clickable card example](https://www.patternfly.org/components/card#clickable), use your keyboard to tab through and try to perform all actions.

**Expected behavior**
To be able to use Tab to navigate cards and enter to click them.

**Comment by thatblindgeye** [comment]
Jessie and I briefly discussed this during a meet, and this may be due to clickable cards currently utilizing visually hidden radio inputs to create the "fully clickable card" effect. Native radio inputs allow tabbing between inputs when no option is selected, but require arrow keys to traverse/select another radio  once a selection has been made. If you Tab to the first clickable card in the example linked above, pressing Down Arrow should navigate to (then trigger) the second card.

**Comment by thatblindgeye** [comment]
Updating this issue to be more of a spike regarding refactoring the markup and API of the selectable/card implementations. This would also require updates made in Core.

**Comment by thatblindgeye** [comment]
Closed by 10859
